changelogs/unreleased/4332-auto-recompile-wait-env-setting.yml

```
@@ -0,0 +1,7 @@
+description: Add environment setting to set the auto-recompile-wait time instead of the config option
+issue-nr: 4332
+change-type: patch
```

This is a breaking change:
```suggestion
change-type: major
```